.github/workflows/matrix-bot.yml

```diff
@@ -49,7 +56,9 @@ jobs:
           message_type: "notice"
 
   abandoned-pr:
-    if: github.event.action == 'closed' && github.event.pull_request.merged == false && github.repository == 'ariel-os/ariel-os'
+    if: |
+      github.event.action == 'closed' && github.event.pull_request.merged == false && github.repository == 'ariel-os/ariel-os' &&
+      !contains(github.even.pull_request.labels.*.name, 'renovate')
```

Comment:
```suggestion
      !contains(github.event.pull_request.labels.*.name, 'renovate')
```

Reply:
Fixed in [550ac2d](https://github.com/ariel-os/ariel-os/pull/1249/commits/550ac2d7051db7dc7c50614cb1dba6715517c07f)